.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+env: # These should match the tox env list
+    - TOXENV=py27
+install: pip install coveralls tox
+script: tox
+# Special snowflake.  Our tests depend on a credentials_secret.json
+before_install:
+    - cp tests/json/credentials.json tests/json/credentials_secret.json
```

Comment:
@mittonk this passes because we only need real credentials when we first record the vcr cassette. Then we just need some filler values to instantiate a client and we replay the recorded requests for the tests.